<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="prevButton">
     <property name="toolTip">
      <string>Previous Playlist</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/tango/images/icons/tango/media-skip-backward.png</normaloff>:/tango/images/icons/tango/media-skip-backward.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="toolTip">
      <string>Next Playlist</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/tango/images/icons/tango/media-skip-forward.png</normaloff>:/tango/images/icons/tango/media-skip-forward.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="playlistCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="toolTip">
      <string>Remove Playlist</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/gnome-icon-theme/images/icons/gnome-icon-theme/close.png</normaloff>:/gnome-icon-theme/images/icons/gnome-icon-theme/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
